onnxruntime/core/providers/rocm/gpu_data_transfer.h

```diff
@@ -5,6 +5,7 @@
 
 #include "rocm_pch.h"
 #include "core/framework/data_transfer.h"
+#include "core/common/common.h"
```

Comment:
Is this include needed ?

Reply:
Can't tell as I do not build locally, but I think it comes from the CI failure.